### ISSUE TITLE
[Image] [Android] Image.getSize doesn't work

### ISSUE THREAD

**Issue by alvaromb** [facebook/react-native#5838]
In Android, using ``Image.getSize`` throws:

```
TaskQueue: Error with task : _reactNative.Image.getSize is not a function
```

Tested under React Native 0.19.0.

**Comment by alvaromb** [comment]
https://productpains.com/post/react-native/image-android-imagegetsize-doesnt-work-in-android/

**Comment by mkonicek** [comment]
Cool, upvoted.

**Comment by sanyuelanv** [comment]
maybe you need [this](https://github.com/facebook/react-native/issues/5838)

**Comment by vivekkhurana** [comment]
Any fix for this?

**Comment by fujinjun** [comment]
0.24.1 also have this problem

**Comment by Proyoyo** [comment]
Having this for android would be handy

**Comment by timzaak** [comment]
0.25.1 does not work

**Comment by hyperh** [comment]
+1 Also does not work for 0.24.1

**Comment by timzaak** [comment]
#7664 and #7690 solve this issue.
please close it.

**Comment by cornedor** [comment]
This has been shipped in 0.28.0

**Comment by ide** [comment]
Closing, Image.getSize is in the latest RN.
